CHANGELOG.md

```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-# v1.0.3
-## Update
+## [Unreleased]
+### Added
+- Tests that verify that the lookup plugin retrieves a Conjur secret into an
+  Ansible variable. 
+  ([cyberark/ansible-conjur-collection#6](https://github.com/cyberark/ansible-conjur-collection/issues/6))
```

Comment:
I don't think we need to announce test enhancements in the CHANGELOG (cc @sgnn7, right?)

see https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#changelog-content

Reply:
This was more for contributors who may be interested that we now have a testing framework but i guess it can be removed